addons/grenades/functions/fnc_flashbangExplosionEH.sqf

```diff
@@ -92,8 +92,9 @@ if (hasInterface && {!isNull ACE_player} && {alive ACE_player}) then {
     _strength = _strength * _losCoefficient;
 
     // Add ace_hearing ear ringing sound effect
-    if (isClass (configFile >> "CfgPatches" >> "ACE_Hearing") && {_strength > 0}) then {
-        [40 * _strength] call EFUNC(hearing,earRinging);
+    if (isClass (configFile >> "CfgPatches" >> "ACE_Hearing") && {_strength > 0 && {EGVAR(hearing,damageCoefficent) > 0.25}}) then {
+        [40 * _strength * EGVAR(hearing,damageCoefficent)] call EFUNC(hearing,earRinging);
```

Comment:
Split calculation into separate variable for readability and logging without double calculations.